executors.go

```diff
@@ -117,6 +212,25 @@ func (c *Connection) Create(model interface{}, excludeColumns ...string) error {
 // ValidateAndUpdate applies validation rules on the given entry, then update it
 // if the validation succeed, excluding the given columns.
 func (c *Connection) ValidateAndUpdate(model interface{}, excludeColumns ...string) (*validate.Errors, error) {
+	v := reflect.Indirect(reflect.ValueOf(model))
```

Comment:
I feel like a lot of this logic is repeated. would make sense to break it out into it's own method? something like this:

```go
type nameMeFunc func(model interface{}, excludeColumns ...string) (*validate.Errors, error)
func nameMe(fn nameMeFunc, model interface{}, excludeColumns ...string)  {
	if v.Kind() == reflect.Slice || v.Kind() == reflect.Array {
 +		var err error
 +		for i := 0; i < v.Len(); i++ {
 +			val := v.Index(i)
 +			if val.Kind() != reflect.Ptr {
 +				val = val.Addr()
 +			}
 +			verrs, err := fn(val.Interface(), excludeColumns...)
 +			if err != nil {
 +				return verrs, err
 +			}
 +			if verrs.HasAny() {
 +				return verrs, nil
 +			}
 +		}
 +		return validate.NewErrors(), err
 +	}
```